extensions/README.md

```diff
@@ -59,6 +59,10 @@ Generally, all new extensions should start out as a beta.
 - Once you've successfully beta tested your new extension, you can open new PR to make your extension live!
 - Simply move the extension's slug out of the beta array and into the production array in `extensions/index.json`.
 
+### Experimental Extensions
+
+We also offer an "experimental" state for extensions. Those extensions will be made available to anyone having the `JETPACK_EXPERIMENTAL_BLOCKS` constant defined in `wp-config.php`.
```

Comment:
I'd include reference here that experimental blocks ship with either this constant or along side beta blocks using the beta constant.

May want to explain experimental blocks are considered production-ready, but are served only to sites requesting it?

Reply:
fixed in 8f12adb4c3c30b7879c1a7057603c509924a80e7